client/src/pages/Challenge.tsx

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import { Container, Row, Col, ButtonToolbar, Button } from "react-bootstrap";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+enum ChallengeDifficulties {
```

Comment:
Since we will be reusing ChallengeDifficulties it would make sense to add this to a types or interfaces folder and import it from there. And then Eren can also import the enum instead of duplicating code

There would be better maintanability as it sets the structure to allow widely used types to be imported instead of duplicating code

Same thing applies to ChallengeDetails if we plan on using that type elsewhere